frontend/src/routes/NotFound.tsx

```diff
@@ -37,7 +37,8 @@ export const NotFound: React.FC<Props> = ({ kind }) => {
                 {t("not-found.url-typo")}
             </p>
             <Trans i18nKey="not-found.actions">
-                foo<Link to="/">bar</Link>
+                You can try visiting <Link to="/">the homepage</Link> or using
```

Comment:
Ok cool. With this change, though, I wonder how many of our `Trans` components really need to be there anymore, now. I think in this changeset this is the only one. The rest could (now) all just be calls to the translation function, right? :thinking:

Reply:
Mhh no. Why do you think we can replace almost all `Trans` uses? We still need them for `Link`s for sure, probably also for `a` unless we want to duplicate the href in all translations. And finally, this "use basic HTML tags in translation" only works with `Trans` and not with `t`. Dunno, maybe it could work with `t` somehow, but ye, not simply like that.

Reply:
Ooh, the last part I didn't know. :shrug: